src/main/kotlin/no/nav/k9/los/tjenester/mock/localSetup.kt

```diff
@@ -53,4 +61,26 @@ object localSetup : KoinComponent {
             }
         }
     }
-}
\ No newline at end of file
+
+    fun initPunsjoppgave() {
```

Comment:
Skal denne fjernes? Kallet til den fra K9Los er kommentert ut

Reply:
Skal ikke fjernes, bruker denne for å få inn en Punsj-oppgave lokalt